service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ConditionLinkSetMapperTest.java

```diff
@@ -49,6 +49,8 @@
 
     private static final String INPUT_JSON_WITH_ACTUAL_PROBLEM_OBSERVATION = CONDITION_FILE_LOCATIONS + "condition_all_included.json";
     private static final String INPUT_JSON_NO_ACTUAL_PROBLEM = CONDITION_FILE_LOCATIONS + "condition_no_problem.json";
+    private static final String INPUT_JSON_NO_ACTUAL_PROBLEM_NO_ONSETDATE = CONDITION_FILE_LOCATIONS + "condition_no_problem_no_onsetdate"
+        + ".json";
```

Comment:
Why this split?

Reply:
Must've been an auto reformat, will fix and check to see if it passes the style checks.